test/cli/require-qunit-test.js

```diff
@@ -0,0 +1,59 @@
+const proxyquire = require( "proxyquire" );
+const resolveStub = {
+	sync( path ) {
+		return path;
+	}
+};
+
+QUnit.module( "requireQUnit", function() {
+	QUnit.test( "finds QUnit in the current working directory", function( assert ) {
+		const localQUnit = {
```

Comment:
No, I don't feel it's particularly needed. I just wanted to see if there was a particular rationale or if it happened organically/unintentionally. Go ahead and merge if you like (or wait for others to review).